index.js

```diff
@@ -11,68 +11,145 @@ module.exports = {
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
 
     // Prefer const over let
-    'prefer-const': [2, {
+    'prefer-const': ['error', {
       'destructuring': 'any',
       'ignoreReadBeforeAssign': false
     }],
 
     // No single if in an "else" block
-    'no-lonely-if': 2,
+    'no-lonely-if': 'error',
 
     // Force curly braces for control flow,
     // including if blocks with a single statement
-    curly: [2, 'all'],
+    curly: ['error', 'all'],
 
     // No async function without await
-    'require-await': 2,
+    'require-await': 'error',
 
     // Force dot notation when possible
-    'dot-notation': 2,
+    'dot-notation': 'error',
 
-    'no-var': 2,
+    'no-var': 'error',
 
     // Force object shorthand where possible
-    'object-shorthand': 2,
+    'object-shorthand': 'error',
 
     // No useless destructuring/importing/exporting renames
-    'no-useless-rename': 2,
+    'no-useless-rename': 'error',
+
+    /**********************/
+    /*   Unicorn Rules    */
+    /**********************/
+
+    // Pass error message when throwing errors
+    'unicorn/error-message': 'error',
+
+    // Uppercase regex escapes
+    'unicorn/escape-case': 'error',
+
+    // Use "new" for most builtins (not String, Number, ..., though)
+    'unicorn/new-for-builtins': 'error',
+
+    // Array.isArray instead of instanceof
+    'unicorn/no-array-instanceof': 'error',
+
+    // for-of instead traditional for loop
+    'unicorn/no-for-loop': 'error',
+
+    // Prevent deprecated `new Buffer()`
+    'unicorn/no-new-buffer': 'error',
+
+    // Keep regex literals safe!
+    'unicorn/no-unsafe-regex': 'off',
+
+    // Use non-fraction numbers if possible, e.g. 1 instead of 1.0
+    'unicorn/no-zero-fractions': 'error',
+
+    // Lowercase number formatting for octal, hex, binary (0x12 instead of 0X12)
+    'unicorn/number-literal-case': 'error',
+
+    // ** instead of Math.pow()
+    'unicorn/prefer-exponentiation-operator': 'error',
 
-    'vue/no-parsing-error': [2, {
+    // includes over indexOf when checking for existence
+    'unicorn/prefer-includes': 'error',
+
+    // append over appendChild
+    'unicorn/prefer-node-append': 'error',
+
+    // remove over removeChild
+    'unicorn/prefer-node-remove': 'error',
+
+    // querySelector instead of getElementBy...
+    'unicorn/prefer-query-selector': 'error',
+
+    // Spread syntax instead of Array.from()
+    'unicorn/prefer-spread': 'error',
+
+    // String methods startsWith/endsWith instead of more complicated stuff
+    'unicorn/prefer-starts-ends-with': 'error',
+
+    // textContent instead of innerText
+    'unicorn/prefer-text-content': 'error',
+
+    // Enforce throwing type error when throwing error while checking typeof
+    'unicorn/prefer-type-error': 'error',
+
+    // Favor regex shorthands (e.g. \d instead of [0-9])
+    'unicorn/regex-shorthand': 'error',
```

Comment:
Noooooooooo, big no from me for this :no_entry: 

I just cant be bothered to remember what eg `w` exactly means so I would always need to have to look it up which means regexes becomes so much less readable (and also due to all the `/` with `\` mixed together.

Reply:
Actually I +1 on this, sometimes readability is better than this weird escape char.

Reply:
I added this rule *because* of readability ☺️ 

When I read a regex and I see `[0-`... I expect a limitation compared to a normal `\d` declaration.

A short example for readability:

`/[0-9]\.[a-zA-Z0-9_]\-[^0-9]/i`
vs.
`/\d\.\w\-\D/i`

---

Of course, if you don't know by heart what `\D` means, it doesn't give you any benefit. But this also applies to every standard function in JavaScript. If you have `[1,2,3].map(x => x**2)` and don't know by heart what `Array.map` does, it's 'easier'/more 'readable' to go for a for loop.

Reply:
Anyway, moving this in my personal preset I think :relaxed: